fdbserver/GrvProxyTransactionTagThrottler.actor.cpp

```diff
@@ -90,7 +90,8 @@ void GrvProxyTransactionTagThrottler::releaseTransactions(double elapsed,
 		uint32_t* numReleased;
 		// Sequence number of the first queued request
 		int64_t nextSeqNo;
-		bool operator<(TagQueueHandle const& rhs) const { return nextSeqNo < rhs.nextSeqNo; }
+		// Reverse order because std::priority_queue is a max heap
+		bool operator<(TagQueueHandle const& rhs) const { return nextSeqNo > rhs.nextSeqNo; }
```

Comment:
This is the wrong way of doing this as it makes it difficult for people to follow the code.

The right way of doing this is to pass `std::greater` to the priority queue.